internal/helm/helm.go

```diff
@@ -0,0 +1,168 @@
+// Package helm contains the epinio-specific core to the helm client libraries. It exposes
+// the functionality to deploy and remove helm charts/releases.
+package helm
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/epinio/epinio/helpers/kubernetes"
+	"github.com/epinio/epinio/internal/duration"
+	"github.com/epinio/epinio/internal/names"
+	"github.com/epinio/epinio/internal/routes"
+	"github.com/epinio/epinio/pkg/api/core/v1/models"
+	"github.com/go-logr/logr"
+	hc "github.com/mittwald/go-helm-client"
+	"github.com/spf13/viper"
+)
+
+const (
+	StandardChart = "https://github.com/epinio/helm-charts/releases/download/epinio-application-0.1.10/epinio-application-0.1.10.tgz"
+	// ChartName:   "/path/to/stable/etcd-operator",			Local directory
+	// ChartName:   "/path/to/stable/etcd-operator.tar.gz",			Local archive
+	// ChartName:   "http://helm.whatever.com/repo/etcd-operator.tar.gz",	Remote archive
+)
+
+type ChartParameters struct {
+	models.AppRef                       // Application: name & namespace
+	Cluster       *kubernetes.Cluster   // Cluster to talk to.
+	Chart         string                // Chart to use for deployment
+	ImageURL      string                // Application Image
+	Username      string                // User causing the (re)deployment
+	Instances     int32                 // Number Of Desired Replicas
+	StageID       string                // Stage ID for ImageURL
```

Comment:
The comment seems wrong. The StageID is only [used as a label on the Deployment](https://github.com/epinio/helm-charts/pull/105/files#diff-778ac2f21904c0d549f62adec97acb398f3544de76019c2f7a485de5848bdb21R27). It doesn't relate to the ImageURL as far as I can tell.

Reply:
This is the id of the staging run which created the image referenced by the image url.
So maybe `id of the staging run which created the image referenced by the image url` is a suitable comment ?

Reply:
or `Stage ID that produced ImageURL` for short

Reply:
Done. 3e786567250304aecd59ee8a0219993c09c4b4d6